{"id":795,"details":"\n  \n  \u003ctable\u003e\n    \u003ctr\u003e\n      \u003ctd\u003eRegion\u003c/td\u003e\n\u003ctd\u003eOccitanie\u003c/td\u003e\n    \u003c/tr\u003e\n    \u003ctr\u003e\n\u003ctd\u003eDepartment\u003c/td\u003e\n\u003ctd\u003eAude\u003c/td\u003e\n\u003c/tr\u003e\n    \n    \u003ctr\u003e\n\u003ctd\u003eReason of Infection\u003c/td\u003e\n\u003ctd\u003econtact with patient\u003c/td\u003e\n\u003c/tr\u003e\n  \u003c/table\u003e\n  ","patient":"Patient From Occitanie, Aude","source":"unknown_source","status":"unreported","confirmed_at":"2020-03-08T00:00:00.000Z","created_at":"2020-03-16T10:13:02.203Z","updated_at":"2020-03-16T10:42:28.833Z","infected_by":"unknown_infection","number":521,"urls":["https://github.com/lperez31/coronavirus-france-dataset"],"recovered_at":null,"gender":"unidentified_gender","age":0,"nationality":"unidentified_nationality","symptomatic_at":null,"slug":"france-case-521-patient-from-occitanie-aude","suggested_url":"france-case-521-patient-from-occitanie-aude","hospital_stay_length_in_days":null,"days_from_symptoms_to_confirmation":null,"without_symptoms":false,"reporting_country":"fr"}